source/common/network/listener_filter_buffer_impl.h

```diff
@@ -47,6 +47,7 @@ class ListenerFilterBufferImpl : public ListenerFilterBuffer, Logger::Loggable<L
   void activateFileEvent(uint32_t events);
   uint64_t capacity() const { return buffer_size_; }
   void resetCapacity(uint64_t size);
+  void setZeroBufferSize(bool zero_buffer_size) { zero_buffer_size_ = zero_buffer_size; }
```

Comment:
I feel we call this method as 'disableOnDataCallback' is better. Otherwise, the method said 'zero buffer size', but actually it is not, it get confused.

Reply:
Thanks for the suggestion, I updated it to disableOnDataCallback